npm/angular/src/mount.ts

```diff
@@ -156,7 +156,7 @@ function bootstrapModule<T> (
   })
 
   // check if the component is a standalone component
-  if ((component as any).ɵcmp.standalone) {
+  if ((component as any).ɵcmp?.standalone) {
```

Comment:
Forgot Angular uses Greek chars, lol

Reply:
I'm pretty sure I had to copy and paste this line cause I don't know how to type `ɵ` on my keyboard 😅